examples/java8/HelloSpring.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * 
```

Comment:
If we're going to have comments, they should be filled out (look at hello.py for reference).  Otherwise it looks sloppy.  We also should add comments to the *.js examples, but that's a separate issue.